src/docs/concepts/timers.adoc

```diff
@@ -63,28 +63,31 @@ Response response = ...
 sample.stop(registry.timer("my.timer", "response", response.status()));
 ----
 
-Note how we don't decide what timer we are accumulating the sample to until it is time to stop the sample. This allows us to dynamically determine certain tags from the end state of the operation we are timing.
+Note how we do not decide the timer to which to accumulate the sample until it is time to stop the sample. This lets us dynamically determine certain tags from the end state of the operation we are timing.
 
 == The `@Timed` annotation
 
-The `micrometer-core` modules contains a `@Timed` annotation that can be used by frameworks to add timing support to either specific types of methods such as those serving web request endpoints or generally to all methods.
+The `micrometer-core` modules contains a `@Timed` annotation that frameworks can use to add timing support to either specific types of methods such as those serving web request endpoints or, more generally, to all methods.
 
 WARNING: Micrometer's Spring Boot configuration does _not_ recognize `@Timed` on arbitrary methods.
 
-An incubating AspectJ aspect is included in `micrometer-core` as well that you can use in your application either through compile/load time AspectJ weaving or through framework facilities that interpret AspectJ aspects and proxy targeted methods in some other way, such as Spring AOP. Here is a sample Spring AOP configuration:
+Also, an incubating AspectJ aspect is included in `micrometer-core`. You can use it in your application either through compile/load time AspectJ weaving or through framework facilities that interpret AspectJ aspects and proxy targeted methods in some other way, such as Spring AOP. Here is a sample Spring AOP configuration:
```

Comment:
Not related to the editing, but we probably shouldn't consider the aspect incubating anymore. @jonatan-ivanov